job-utilities/local-development/README.MD

```diff
@@ -0,0 +1,13 @@
+# local development job-utilities
+
+Python utilities to help with local development. Currently there is just a program for scheduling jobs to run locally
+
+## local_scheduler
+
+This program takes the job details from the job-details.json file and maps them to a schedule using the schedule Python library.
+Cron jobs are assumed to be run once every day at a given time. Most jobs are likely to be interval based, which run every given period of time.
+
+### Running
+
+Ensure you have the right dependencies by running `pip install -r requirements.txt` and then simply run the local_scheduler.py program.
```

Comment:
Shouldn't this be part of the docker image, so that developers don't have to do this?

Reply:
don't currently have a docker image for the local scheduling program, but that is a good idea I should make one

Reply:
Will do further work on the local scheduler in a later ticket